Automattic/wp-calypso
# AI Logo Generator: UI adjustments

Closes https://github.com/Automattic/jetpack/issues/34376
Closes https://github.com/Automattic/wp-calypso/issues/85859
Closes https://github.com/Automattic/jetpack/issues/34379

## Proposed Changes

* Adjustments to colors, spacing and mobile
* Applies translation function calls
* Closes the modal and reloads the page after  applying a new logo
* Uses the store to disable actions when saving or applying a logo

## Testing Instructions



* Go to the generator modal
* Generate a logo
* Check that the colors are correct, using the theme variables
* Save the logo. Check that other buttons are disabled while saving is in progress
* Generate another logo
* Without saving it first, use it on the website
* Check that it is first saved, then applied, with buttons disabled while this is in progress
* Check that the modal is closed and the window is refreshed
* Check that the new logo is loaded and displayed correctly
* Check the modal style in a mobile viewport

## Pre-merge Checklist
